src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -377,7 +377,10 @@ public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out CustomNodeWork
 
         public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out ICustomNodeWorkspaceModel ws)
         {
-            return TryGetFunctionWorkspace(id, isTestMode, out ws);
+            CustomNodeWorkspaceModel workSpace;
```

Comment:
I found this bug before task was created. But Since task is created... Add link to it: [MAGN-8022](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8022) Publish Current Workspace for a custom node hangs indefinitely